gdal/frmts/rda/rdadataset.cpp

```diff
@@ -1688,8 +1699,21 @@ void GDALRDADataset::BatchFetch(int nXOff, int nYOff, int nXSize, int nYSize)
 
         for( int j = 0; j < nToDownload; j++ )
         {
-            if( pasResults[j]->pabyData )
+            if( pasResults[j]->pszErrBuf != nullptr )
+            {
+                CPLString url = apszURLLists[i+j];
+                CPLString message = pasResults[j]->pabyData ?
+                                    CPLSPrintf("%s: %s", pasResults[j]->pszErrBuf,
+                                               reinterpret_cast<const char*>(pasResults[j]->pabyData )) :
+                pasResults[j]->pszErrBuf;
```

Comment:
should be intended at the same level as the above CPLSPrintf(), which could be just 4 characters right of the CPLString message declaration

Reply:
replaced